File: angular-web2-app/src/app/entities/bezbednosni-dokument/bezbednosni-dokument.ts
export class BezbednosniDokument {
    idBezbednosnogDokumenta:number;
    Tip:string;
    Status:string;
    //idKorisnika:number;
    Ekipa:string;
    Detalji:string;
    Beleske:string;
    TelefonskiBroj:string;
    DatumKreiranja:Date;
    AllWorkOperationsCompleted:boolean;
    AllTagsRemoved:boolean;
    GroundingRemoved:boolean;
    ReadyForService:boolean;

    constructor(idBezbednosnogDokumenta:number, Tip:string, Status:string, Ekipa:string, Detalji:string, Beleske:string,
        TelefonskiBroj:string, DatumKreiranja:Date, AllWorkOperationsCompleted:boolean, AllTagsRemoved:boolean, GroundingRemoved:boolean, ReadyForService:boolean) {
            this.idBezbednosnogDokumenta = idBezbednosnogDokumenta;
            this.Tip = Tip;
            this.Status = Status;
            this.Ekipa = Ekipa;
            this.Detalji = Detalji;
            this.Beleske = Beleske;
            this.TelefonskiBroj = TelefonskiBroj;
            this.DatumKreiranja = DatumKreiranja;
            this.AllWorkOperationsCompleted = AllWorkOperationsCompleted;
            this.AllTagsRemoved = AllTagsRemoved;
            this.GroundingRemoved = GroundingRemoved;
            this.ReadyForService = ReadyForService;
    }
}

File: api/webAPI/WebApplication1/Models/BezbednosniDokument.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Models
{
    public class BezbednosniDokument
    {
        public int idBezbednosnogDokumenta { get; set; }
        public string Tip { get; set; }
        public string Status { get; set; }
        //public int idKorisnika { get; set; }
        public string Ekipa { get; set; }
        public string Detalji { get; set; }
        public string Beleske { get; set; }
        public string TelefonskiBroj { get; set; }
        public DateTime DatumKreiranja { get; set; }
        public bool AllWorkOperationsCompleted { get; set; }
        public bool AllTagsRemoved { get; set; }
        public bool GroundingRemoved { get; set; }
        public bool ReadyForService { get; set; }
    }
}